src/Publish/Microsoft.NET.Sdk.Publish.Tasks/WebConfigTransform.cs

```diff
@@ -150,7 +149,7 @@ private static void TransformAspNetCore(XElement aspNetCoreElement, string appNa
             // Set the hostingmodel attribute only if it is not already set in the web.config and AspNetCoreHostingModel property is set.
             if (hostingModelAttributeValue == null && !string.IsNullOrEmpty(aspNetCoreHostingModel))
             {
-                switch(aspNetCoreHostingModel.ToLower())
+                switch(aspNetCoreHostingModel.ToLowerInvariant())
```

Comment:
ToUpperInvariant is suggested to be used instead of ToLowerInvariant

Reply:
ya sure.